### ISSUE TITLE
visibbility chain

### ISSUE THREAD

**Issue by Indahjelita** [chainapsis/keplr-chain-registry#636]
Helichain goes mainet but chain cant add to keplar?

**Comment by claraexmachina** [comment]
for non-native chains you need to first create a PR to this repo with the necessary information (refer to the readme of it). Once it is merged it will appear on chains.keplr.app.

**Comment by claraexmachina** [comment]
Hi. We will temporarily close this issue due to lack of information about it. Please open a new issue if it still persists. Thank you for your understanding!
